### ISSUE TITLE
Does this work if i leave developer mode after?

### ISSUE THREAD

**Issue by doyourclanwars** [MercuryWorkshop/Pollen#15]
sorry i dont really know how this works

**Comment by flebedev77** [comment]
no, the original policies will be restored if you leave developer mode.

**Comment by doyourclanwars** [comment]
does this still even work in the latest version? I tried it and nothing seems to happen

**Comment by flebedev77** [comment]
did you follow the instructions correctly? any errors?

**Comment by doyourclanwars** [comment]
it works perfectly fine, but the policies dont change at all. 
I am using a chromebook with 2 users though, and the user with policies arent the main user

**Comment by Charlieis1nsan3** [comment]
to use this exploit you have to enable developer mode. once that is done. you want to go into your home screen of your chromeos device and put control+alt+ left arrow or right arrow. input the commands that they mentioned earlier and boom.  your editring your policies
